documentation/en/faqs.md

```diff
@@ -0,0 +1,110 @@
+More information about Lotus can be found [here](https://lotu.sh/).
+
+#### What are the components of Lotus?
+
+Lotus is composed of two separate pieces that can talk to each other:
+
+The Lotus Node can sync the blockchain, validating all blocks, transfers, and deals
+along the way. It can also facilitate the creation of new storage deals. If you are not 
+interested in providing your own storage to the network, and do not want to produce blocks
+yourself, then the Lotus Node is all you need!
+
+The Lotus Storage Miner does everything you need for the registration of storage, and the
+production of new blocks. The Lotus Storage Miner communicates with the network
+by talking to a Lotus Node over the JSON-RPC API.
+
+## Setting up a Lotus node
+
+#### How do I set up a Lotus node?
+
+Follow the instructions found [here](https://docs.lotu.sh/en+getting-started).
+
+#### Where can I get the latest version of Lotus?
+
+Download the binary tagged as the `Latest Release` from the
+ [Lotus Github repo](https://github.com/filecoin-project/lotus/releases).
+ 
+#### What operating systems can Lotus run on?
+
+Lotus can build and run on most Linux and MacOS systems with at least 
+8GB of RAM. Windows is not yet supported.
+
+#### How can I update to the latest version of Lotus?
+
+To update Lotus, follow the instructions [here](https://lotu.sh/en+updating-lotus).
+
+#### How do I prepare a fresh installation of Lotus?
+
+Stop the Lotus daemon, and delete all related files, including sealed and chain data by 
+running `rm ~/.lotus ~/.lotusstorage`.
+
+Then, install Lotus afresh by following the instructions 
+found [here](https://docs.lotu.sh/en+getting-started).
+
+## The Test Network
+
+#### What is the test network?
+
+The test network, sometimes called Testnet, is a live network of Lotus nodes run by the community. It has 2 PiB  
+of storage (and growing!) dedicated to it.
+
+#### Is FIL on the test network worth anything?
+
+Nothing at all! Real-world incentives may be provided in a future phase of the test network, but this is 
+yet to be confirmed.
+
+#### Will there be future phases of the test network?
+
+Yes, there will be at least one more phase of the test network. We plan on introducing interoperable
+[go-filecoin nodes](https://github.com/filecoin-project/go-filecoin#filecoin-go-filecoin)
+in a future phase.
+
+#### How can I see the status of the test network?
+
+The [dashboard](https://stats.testnet.filecoin.io/) displays the status of the network as well as ton
+of other metrics you might find interesting.
+
+## Mining with a Lotus node on the test network
+
+#### How do I get started mining with Lotus?
+
+Follow the instructions found [here](https://lotu.sh/en+mining).
+
+#### What are the minimum hardware requirements?
+
+An example test configuration, and minimum hardware requirements can be found 
+[here](https://lotu.sh/en+hardware-mining). 
+
+Note that these might NOT be the minimum requirements for mining on the real main network.
```

Comment:
I would use Mainnet terminology consistently here instead of `real main network`